settings/DevHome.Settings/Strings/en-us/Resources.resw

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 

```

Comment:
This is adding whitespaces. Please remove.

Reply:
Removed.